README.md

```diff
@@ -28,19 +28,22 @@ NeptuneUI and QtApplicationManager.
 
 ### Using vagrant
 
+Please note that we only run this setup in a GNU/Linux system at Pelagicore. It
+should still work under Windows or OSX, we haven't tried it.
+
 Dependencies:
 
 * Vagrant
-* Docker or VirtualBox
-* virtualization enabled in bios
+* Docker
+* Virtualization enabled in BIOS
 
 Procedure:
 
 1. Clone the pelux-manifests git repository.
 2. Start vagrant
 
     ```bash
-    vagrant up --provider="docker"
+    vagrant up
```

Comment:
Did you verify that not specifying provider works? I have had problem with it

Reply:
I never used the --provider flag and it always worked for me, when did you have problems with that?

Reply:
isn't the default provider specified from the base-box ? what's the base-box using ?

Reply:
Yes, I tried it without specifying provider.

Reply:
@rpannek it was some time during the summer I think